client/components/LayoutEditor/LayoutEditorPagesCollections.tsx

```diff
@@ -84,28 +108,44 @@ const LayoutEditorPagesCollections = (props: Props) => {
 					value={content.title}
 					onChange={(evt) => setTitle(evt.target.value)}
 				/>
-				<FormGroup label="Collections & Pages">
-					<Popover
-						aria-label="Choose pinned Pubs for this block"
-						className="order-picker-popover"
-						placement="bottom-end"
-						content={
-							<OrderPicker
-								availableItems={availableItems}
-								selectedItems={selectedItems}
-								onSelectedItems={setSelectedItems}
-								renderItem={(item, handleClick) => (
-									<PubMenuItem title={item.title} onClick={handleClick} />
-								)}
+
+				<div className="dropdown-grouping">
+					<div className="dropdown">
```

Comment:
I think I found a fix for the way this is jumping around as the `MenuSelect` width changes — if you add:
```tsx
buttonProps={{fill: true, alignText: 'left'}}
```
to the `MenuSelect`, it'll cause the `Button` that it renders to (1) fill the width of the container and (2) push its text to the left and the caret to the right. (Eventually we may find that this should be the default behavior). By the way, I'm reading this off the [Blueprint docs](https://blueprintjs.com/docs/#core/components/button) which are extremely handy.

With this done all that's left is to set the width or min-width of this div to something like 150px. I'd suggest giving it a new class name and referencing it in `layoutEditor.scss` (nested under `.layout-editor-pages-component`)